tests/js/autoAdjustTests.js

```diff
@@ -0,0 +1,94 @@
+/*!
+Copyright 2015 CERTH
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($) {
+    "use strict";
+    
+    fluid.registerNamespace("gpii.tests");
+
+    fluid.defaults("gpii.tests.autoAdjustControls", {
+        gradeNames: ["gpii.adjuster.autoAdjust", "autoInit"],
+        model: {
+            autoAdjust: false
+        }
+    });
+    
+    fluid.defaults("gpii.tests.autoAdjust", {
+        gradeNames: ["fluid.test.testEnvironment", "autoInit"],
+        components: {
+            autoAdjuster: {
+                type: "gpii.tests.autoAdjustControls",
+                container: ".gpii-onOffSwitch-container",
+                options: {
+                    strings: {
+                        label: "Auto-adjust",
+                        description: "Preferences will be automatically adjusted based on background noise and ambient light levels.",
+                    }
+                }
+            },
+            autoAdjustTester: {
+                type: "gpii.tests.autoAdjustTester"
+            }
+        }
+    });
+
+    fluid.defaults("gpii.tests.autoAdjustTester", {
+        gradeNames: ["fluid.test.testCaseHolder", "autoInit"],
+        testOptions: {
+            defaultInputStatus: false,
+            newValue: true
+        },
+        modules: [{
+            name: "Auto-adjust adjuster",
+            tests: [{
+                name: "Description text",
+                func: "gpii.tests.autoAdjust.validateText",
+                args: ["{autoAdjuster}.dom.autoAdjustDescription", "{autoAdjuster}.options.strings.description"]
+            }, {
+                name: "Validate autoAdjust",
+                expect: 4,
+                sequence: [{
```

Comment:
These don't need to be in a sequence. Sequence's allow for cases where you make a change with a function and then do the test in an event listener.